lib/taskana-core/src/main/java/pro/taskana/impl/TaskQueryImpl.java

```diff
@@ -1014,14 +1014,16 @@ private void setupAccessIds() {
             this.addOrderCriteria(columnName.toString(), sortDirection);
             checkOpenAndReadPermissionForSpecifiedWorkbaskets();
             setupAccessIds();
+
             if (columnName.equals(TaskQueryColumnName.CLASSIFICATION_NAME)) {
                 joinWithClassifications = true;
-                addClassificationNameToSelectClauseForOrdering = true;
             }
-            if (columnName.equals(TaskQueryColumnName.A_CLASSIFICATION_NAME)) {
-                joinWithAttachments = true;
+            List<TaskQueryColumnName> attachmentColumns = Arrays.asList(
```

Comment:
I would prefer it to select them by convention like all columns starting with A_
You will not adapt this while adding a new column to the enum.
furthermore this is a static value which could be extracted. I would actually include it in the enum class itself.

Reply:
Good point!
Also didn't liked my solution at this point, but I wasn't aware that I could add such a function to a Enum class.
Thanks for reporting!